languages/org.xtext.builddsl/src-gen/org/xtext/builddsl/build/impl/BuildPackageImpl.java

```diff
@@ -108,6 +108,7 @@ public static BuildPackage init()
     isInited = true;
 
     // Initialize simple dependencies
+    TypesPackage.eINSTANCE.eClass();
```

Comment:
@cdietrich We should also remove examples/org.xtext.template.examples/xtend-gen 
I think it's old and was moved to .xtend-gen